amr-wind/wind_energy/actuator/FLLC.H

```diff
@@ -33,6 +33,13 @@ struct FLLCData
     bool different_sizes;
     RealList span_distance_vel;
     RealList span_distance_force;
+
+    // non-uniform variables
+    bool nonu{true};    // non-uniform flag
+    amrex::Real eps_dr; // the ratio of epsilon to actuator width
+    RealList r_n;       // non-uniform radius
+    RealList dr_n;       // non-uniform radius
+    VecList lift_n;     // non-uniform lift
```

Comment:
```suggestion
    RealList nonuniform_r;
    RealList nonuniform_dr; 
    VecList nonuniform_lift; 
```
Same thing here. This is much clearer when we get to the actual math. Consider your question to me on slack vs my reply
``` c++
// question
eps =  interp::linear(r_, data.span_distance_vel, data.optimal_epsilon); (edited) 

// reply
eps_nonuniform = interp::linear(r_uniform, eps_optimal, r_nonuniform) 
```

The first one it is hard to interpret and check because of the names. The second is much cleaner b/c of the names.

Reply:
Agreed, fixed in latest version.